<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>switcher</class>
 <widget class="QMainWindow" name="switcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <widget class="QWidget" name="">
          <property name="geometry">
           <rect>
            <x>473</x>
            <y>100</y>
            <width>231</width>
            <height>239</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>aleft</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>atop</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>awidth</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>aheigth</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLineEdit" name="targer_aleft">
                 <property name="text">
                  <string>10.0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="targer_atop">
                 <property name="text">
                  <string>20.0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="targer_awidth">
                 <property name="text">
                  <string>80.0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="targer_aheigth">
                 <property name="text">
                  <string>60.0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QWidget" name="">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>bleft</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>btop</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>bwidth</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>bheigth</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLineEdit" name="source_bleft">
                 <property name="text">
                  <string>0.0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="source_btop">
                 <property name="text">
                  <string>0.0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="source_bwidth">
                 <property name="text">
                  <string>70.0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="source_bheigth">
                 <property name="text">
                  <string>40.0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="create">
             <property name="text">
              <string>Документ</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>736</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
